middleware/instrument.ts

```diff
@@ -0,0 +1,12 @@
+import dotenv from 'dotenv'
+import * as Sentry from '@sentry/node'
+import { nodeProfilingIntegration } from '@sentry/profiling-node'
+
+dotenv.config()
+
+Sentry.init({
+  dsn: `https://${process.env.SENTRY_PUBLIC_KEY}@${process.env.SENTRY_HOST}.ingest.de.sentry.io/${process.env.SENTRY_PROJECT_ID}`,
```

Comment:
I think it would be better if you used destructuring to get constants from process.env 
`const {SENTRY_HOST, ...} = process.env;`